docs/user/gcp/iam.md

```diff
@@ -0,0 +1,45 @@
+# GCP Service Account
+
+Before proceeding with the OpenShift install, you should create a secondary IAM service account following the steps
+outlined here:
+
+
+## Step 1: Create a Service Account
+
+Create a GCP service account using the console or the CLI
+
+[GCP: Creating a service account][sa-create]
+
+## Step 2: Assign Project Roles to your Service Account
+
+You need to assign the proper roles to the newly created service account so it can be used to create an OpenShift cluster.
+The minimum set of roles you can assign the service account are the following:
+- Compute Admin
```

Comment:
link to `https://cloud.google.com/iam/docs/understanding-roles#predefined_roles` maybe??